### PR TITLE
Fixed major bug

### PR DESCRIPTION
**Pull request by ErikBjare**
Can't believe we missed this @johan-bjareholt :rofl: 

:man_facepalming:

### CODE REVIEW

**Comment by ErikBjare** [comment]
I'm going to start using pylint since that would have caught it...

**Comment by ErikBjare** [comment]
I think this would need to be merged first in order for the behavior to be correct: https://github.com/ActivityWatch/aw-core/pull/59

File: aw_watcher_afk/afk.py [hunk 1]
```diff
@@ -26,40 +26,43 @@
 
 class Settings:
     def __init__(self, config_section):
+        # Time without input before we're considering the user as AFK
         self.timeout = config_section.getfloat("timeout")
-        self.update_time = config_section.getfloat("update_time")
```
Comment (johan-bjareholt) [inline aw_watcher_afk/afk.py]: Now when we have client-side heartbeat merging we can remove update_time altogether in aw-watcher-afk.

**Comment by ErikBjare** [comment]
The AFK watcher occasionally hickups (even with these fixes) that leads to the occasional:

```
Received heartbeat after pulse window, inserting as new event.
```

This is likely because `pulsetime` isn't set correctly in some cases.

File: aw_watcher_afk/afk.py (hunk 1 above)
Comment (ErikBjare) [reply aw_watcher_afk/afk.py]: Done!

**Pull request merged**